airbyte-integrations/connectors/source-chargify/source_chargify/schemas/customers.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "first_name": {
+      "type": ["string"]
+    },
+    "last_name": {
+      "type": ["string"]
+    },
+    "email": {
+      "type": ["string"],
+      "format": "date-time"
```

Comment:
Are you sure the email address has `date-time` format 🤔

Reply:
yeah forget about it! I recreate the schemas